src/prefect/cli/server.py

```diff
@@ -90,8 +91,7 @@ def server():
     "--version",
     "-v",
     help="The server image versions to use (for example, '0.10.0' or 'master')",
-    # TODO: update this default to use prefect.__version__ logic
-    default="latest",
+    default="master" if len(prefect.__version__.split("+")) > 1 else "latest",
```

Comment:
Just looking to the future, I think we might want to default to `prefect.__version__` instead of `"latest"`

Reply:
Ah yeah good call